src/components/dev-hub/survey-banner.js

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react';
+import styled from '@emotion/styled';
+import Button from './button';
+import Icon from '@leafygreen-ui/icon';
+import { fontSize, screenSize, size, colorMap } from './theme';
+
+const Footer = styled('div')`
+    display: ${({ showSurvey }) => (showSurvey ? `block` : `none`)};
+    background-color: ${colorMap.greyDarkTwo};
+    position: fixed;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    border: 1px solid ${colorMap.greyDarkOne};
+`;
+
+const FooterContent = styled('div')`
+    display: flex;
+    justify-content: space-between;
+    @media ${screenSize.mediumAndUp} {
+        margin: 0 2% 0 15%;
+    } ;
+`;
+
+const SurveyContent = styled('div')`
+    @media ${screenSize.mediumAndUp} {
+        display: flex;
+    }
+`;
+
+const SurveyText = styled('div')`
+    font-family: 'Fira Mono', monospace;
+    font-size: ${fontSize.tiny};
+    padding: 18px;
+    @media ${screenSize.upToMedium} {
+        padding: ${size.xsmall} ${size.xsmall} 0px ${size.xsmall};
```

Comment:
```suggestion
        padding: ${size.xsmall} ${size.xsmall} 0px;
```

This is a neat small shorthand for doing the same thing!